packages/angular/projects/tx-native-angular-sdk/tests/T.decorator.spec.ts

```diff
@@ -1,77 +1,55 @@
-/* eslint-disable prefer-arrow/prefer-arrow-functions */
-import { Component, OnInit } from '@angular/core';
-import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { Component } from '@angular/core';
+import { TestBed } from '@angular/core/testing';
 import { tx } from '@transifex/native';
 
 import { T } from '../src/lib/T.decorator';
 
-
 describe('T Decorator', () => {
-  const instanceConfig = {
-    token: 'token',
-    alias: 'alias',
-    controlled: false,
-  };
-
   @Component({
-    selector: 'test-cmp',
-    template: '<div>{{testProperty}}</div>',
-    styles: [],
+    template: '<div>{{ testProperty }}</div>',
   })
-  class TestComponent implements OnInit {
-    @T('not-trans-dec', { _key: 'test' })
-      testProperty: any;
-
-    ngOnInit() {
-    }
+  class TestComponent {
+    @T('not-trans-dec', { _key: 'test' }) testProperty!: string;
   }
 
   @Component({
-    selector: 'test-cmp',
-    template: '<div>{{testProperty}}</div>',
-    styles: [],
+    template: '<div>{{ testProperty }}</div>',
   })
-  class TestWithInstanceComponent implements OnInit {
-    @T('not-trans-dec', { _key: 'test' }, instanceConfig)
-      testProperty: any;
-
-    ngOnInit() {
-    }
+  class TestWithInstanceComponent {
+    @T('not-trans-dec', { _key: 'test' }, 'alias') testProperty!: string;
   }
 
-  let component: TestComponent;
-  let fixture: ComponentFixture<TestComponent>;
-
-  let componentWithInstance: TestWithInstanceComponent;
-  let fixtureWithInstance: ComponentFixture<TestWithInstanceComponent>;
-
   beforeEach(() => {
-    spyOn(tx, 'translate').and.returnValue('ok-translated-dec');
+    TestBed.configureTestingModule({ teardown: { destroyAfterEach: false } });
   });
 
   beforeEach(() => {
-    fixture = TestBed.createComponent(TestComponent);
-    component = fixture.componentInstance;
-    fixture.detectChanges();
-
-    fixtureWithInstance = TestBed.createComponent(TestWithInstanceComponent);
-    componentWithInstance = fixtureWithInstance.componentInstance;
-    fixtureWithInstance.detectChanges();
+    spyOn(tx, 'translate').and.returnValue('ok-translated-dec');
   });
 
   it('should test the decorator T', () => {
-    const compiled = fixture.debugElement.nativeElement;
-    expect(tx.translate).toHaveBeenCalled();
-    expect((compiled as HTMLDivElement).innerHTML)
-      .toContain('ok-translated-dec');
+    const fixture = TestBed.createComponent(TestComponent);
+    const component = fixture.componentInstance;
+
+    fixture.detectChanges();
+    expect(tx.translate).toHaveBeenCalledTimes(2); // Why twice?
```

Comment:
I think we can ignore this, and use a simple `hasBeenCalled()` . I couldn't find any reason why it's called twice... probably some internal mechanism for decorators? Let's assume that is working fine.